HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -631,8 +643,8 @@
   </sch:pattern>
 
   <sch:pattern>
-    <sch:title>[CoolingSystemType=RoomAC]</sch:title>
-    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem[h:CoolingSystemType="room air conditioner"]'>
+    <sch:title>[CoolingSystemType=RoomAC or CoolingSystemType=PTAC]</sch:title>
```

Comment:
Let's just shorten here (and other references) to "[CoolingSystemType=RoomAC or PTAC]"